BetterShards/pom.xml

```diff
@@ -1,101 +0,0 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0"
-  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
-                      http://maven.apache.org/xsd/maven-4.0.0.xsd">
-<modelVersion>4.0.0</modelVersion>
-	<groupId>vg.civcraft.mc.bettershards</groupId>
-	<artifactId>BetterShards</artifactId>
-	<packaging>jar</packaging>
-	<version>1.4.26</version>
-	<name>BetterShards</name>
-	<url>https://github.com/Civcraft/BetterShards</url>		  
-	
-	<dependencies>
-		<dependency>
-			<groupId>vg.civcraft.mc.bettershards</groupId>
-			<artifactId>BetterShardsBukkit</artifactId>
-			<version>1.0.0</version>
-		</dependency>
-	</dependencies>
-	
-	<build>
-		<defaultGoal>clean package</defaultGoal>
-		<directory>../target</directory> <!-- Build it one level up, because it is the one we care about -->
-		<finalName>BetterShards-${project.version}</finalName>
-		<resources>
-			<resource>
-				<targetPath>.</targetPath>
-				<filtering>true</filtering>
-				<directory>..</directory>
-				<includes>
-					<include>LICENSE.txt</include>
-				</includes>
-			</resource>
-			<!-- plugin.yml here to fill in the correct content. -->
-			<resource>
-				<targetPath>.</targetPath>
-				<filtering>true</filtering>
-				<directory>../BetterShardsBukkit</directory>
-				<includes>
-					<include>plugin.yml</include>
-				</includes>
-			</resource>
-			<resource>
-				<targetPath>.</targetPath>
-				<filtering>true</filtering>
-				<directory>../BetterShardsBungee</directory>
-				<includes>
-					<include>bungee.yml</include>
-				</includes>
-			</resource>
-			<!-- config.yml needs its own location so it does not mess with the bukkit plugin. -->
-			<resource>
-				<targetPath>./bungee</targetPath>
```

Comment:
@Rourke750 For a better way to do it, look at https://github.com/ProgrammerDan/SkilUp

I didn't look close enough here to see if Max is already doing it.

But basically in the SkilUp-light refactor I have separate config.yml for bungee and bukkit, there is no conflict and no special jar resource handling necessary as bukkit doesn't shade in bungee or vice versa. They both shade in common class files but no common resources.